src/pages/PoolsOverview.tsx

```diff
@@ -1,5 +1,42 @@
 import { Layout } from "@stellar/design-system";
+import { Breadcrumbs } from "components/Breadcrumbs";
 
 export const PoolsOverview = () => (
-  <Layout.Inset>Pools overview page</Layout.Inset>
+  <Layout.Inset>
+    Pools overview page
+    {/* ./overview/top-liquidity-positions */}
+    <div style={{ marginTop: 20 }}>
+      <Breadcrumbs
+        links={[
+          {
+            link: "./overview",
+            text: "Overview",
+          },
+          {
+            link: "./overview/top-liquidity-positions",
+            text: "Top Liquidity Positions",
+          },
+        ]}
```

Comment:
We'll need to figure out how to best split the links (with text labels), but that is out of the scope of this component.